### PR TITLE
Update mirrordisplays.rb

### PR DESCRIPTION
**Pull request by leoj3n**
Fix download location

### CODE REVIEW

File: Casks/mirrordisplays.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   version :latest
   sha256 :no_check
 
-  url 'http://www.fabiancanas.com/downloads/MirrorDisplays.zip',
+  url 'https://github.com/fcanas/mirror-displays/releases/download/1.0/MirrorDisplays.zip',
```
Comment (vitorgalvao) [inline Casks/mirrordisplays.rb]: Needs a `# github.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

Also, please add a `appcast 'https://github.com/fcanas/mirror-displays/releases.atom' line under.

**Comment by leoj3n** [comment]
Ok @vitorgalvao should be ready now but I did it through the web interface so don't think I can squash without downloading the patch branch locally which I'm not able to do right now.

File: Casks/mirrordisplays.rb [hunk 2]
```diff
@@ -2,10 +2,12 @@
   version :latest
   sha256 :no_check
 
-  url 'http://www.fabiancanas.com/downloads/MirrorDisplays.zip',
+  # github.com is the official download host per the vendor homepage
+  url 'https://github.com/fcanas/mirror-displays/releases/download/1.0/MirrorDisplays.zip',
     :referer => 'http://www.fabiancanas.com/Projects/MirrorDisplays'
```
Comment (vitorgalvao) [inline Casks/mirrordisplays.rb]: Just noticed, this shouldn’t be needed, anymore.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 8001bb1d9658c49dde0efba7dcbed79e6c343780. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**